opa.yaml

```diff
@@ -0,0 +1,51 @@
+package:
+  name: opa
+  version: 0.64.1
+  epoch: 0
+  description: Open Policy Agent (OPA) is an open source, general-purpose policy engine.
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - go
+      - wolfi-baselayout
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/open-policy-agent/opa
+      tag: v${{package.version}}
+      expected-commit: 298f97d79da33ede2bd9ec9cbc5641daa63a8587
+
+  - uses: go/build
+    with:
+      modroot: .
+      packages: .
+      output: opa
+
+  - uses: strip
+
+update:
+  enabled: true
+  github:
+    identifier: open-policy-agent/opa
+    strip-prefix: v
+    use-tag: true
+
+subpackages:
+  - name: ${{package.name}}-compat
+    description: Opa binary symlink for Neuvector
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.contextdir}}/usr/local/bin
+          ln -s /usr/bin/opa ${{targets.contextdir}}/usr/local/bin/opa
```

Comment:
```suggestion
```

Let's ditch this and create a symlink over in the main neuvector package(s) that depend on opa.

Reply:
Works for me - it's just not needed compact packages otherwise.